documentation/sphinx/source/disk-snapshot-backup.rst

```diff
@@ -0,0 +1,90 @@
+
+.. _disk-snapshot-backups:
+
+#################################
+Disk Snapshot Backup and Restore
+#################################
+
+This document covers disk snapshot based backup and restoration of a FoundationDB database. This tool leverages disk level snapshots and gets a point-in-time consistent copy of the database. The disk snapshot backup can be used to provide additional level of protection in case of hardware or software failures, test and development purposes or for compliance reasons.
+
+.. _disk-snapshot-backup-introduction:
+
+Introduction
+============
+
+FoundationDB's disk snapshot backup tool makes a consistent, point-in-time backup of FoundationDB database without downtime by taking crash consitent snapshot of all the disk stores that has persistent data.
+
+Crash consitent snapshot feature at file-system or disk level is a pre-requsiste for using this feature.
+
+Disk snapshot backup tool orchestrates the snapshotting of all the disk images and ensures that they are restorable in a point-in-time consistent basis. Externally, all the disk stores of a cluster could be snapshotted, but those disk snapshots will not be point-in-time consistent and hence not restorable.
```

Comment:
` Externally, all the disk stores of a cluster could be snapshotted, but those disk snapshots will not be point-in-time consistent and hence not restorable.`
can be changed to 
`The disk snapshot backup tool must be used to create the disk snapshots because disk snapshots requires to be point-in-time consistent to be restorable, which can be achieved by the off-the-shelf disk snapshot tools`

Reply:
I feel the second line is a bit redundant and got rid of it.